ui/components/multichain/address-copy-button/address-copy-button.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ICON_NAMES, ButtonBase } from '../../component-library';
+import {
+  BackgroundColor,
+  IconColor,
+  TextVariant,
+  TextColor,
+  Size,
+  BorderRadius,
+  AlignItems,
+  OVERFLOW_WRAP,
+} from '../../../helpers/constants/design-system';
+import { useCopyToClipboard } from '../../../hooks/useCopyToClipboard';
+import { shortenAddress } from '../../../helpers/utils/util';
+import Tooltip from '../../ui/tooltip/tooltip';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+export const AddressCopyButton = ({
+  address,
+  shorten = false,
+  wrap = false,
+}) => {
+  const displayAddress = shorten ? shortenAddress(address) : address;
+  const [copied, handleCopy] = useCopyToClipboard();
+  const t = useI18nContext();
+
+  return (
+    <Tooltip position="bottom" title={copied ? t('copiedExclamation') : null}>
+      <ButtonBase
+        backgroundColor={BackgroundColor.primaryMuted}
+        onClick={() => handleCopy(address)}
+        padding={0}
```

Comment:
There is no vertical padding on button base so you can remove this
```suggestion
```

Reply:
Makes sense!  Fixed!